docs/develop/python/message-passing.mdx

```diff
@@ -310,10 +310,10 @@ To obtain an Update handle, you can:
 - Use [`start_update`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#start_update) to start an Update and return the handle, as shown in the preceding example.
 - Use [`get_update_handle_for`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#get_update_handle_for) to fetch a handle for an in-progress Update using the Update ID and Workflow ID.
 
-:::info NON-TYPE SAFE API CALLS
+:::info SEND MESSAGES WITHOUT TYPE SAFETY
 
 In real-world development, sometimes you may be unable to import Workflow Definition method signatures.
-When you don't have access to the Workflow Definition or it isn't written in Python, you can still use non-type safe APIs and dynamic method invocation.
+When you don't have access to the Workflow Definition or it isn't written in Python, you can still use APIs that aren't type-safe, and dynamic method invocation.
```

Comment:
I think this could be stronger by motivating why you don't have access and making it more general so it can be re-purposed more easily. For example when the Workflow Definition is written in another language (vs Python specifically), _so it's not possible to perform compile-time type safety checks_.

Additional free form thoughts here you can consider mining for greater reader clarity:

APIs built with string-based method invocation are flexible and allow dynamic method calls across different programming languages, but because they rely on strings constants, they lack compile-time type safety checks. This makes it harder to catch errors through your tooling/processes (such as mistyped method names or parameters mismatches) until runtime, which is not when you want to be finding these issues.

Temporal's dynamic invocation system adds a level of reliability to an inherently unsafe process by allowing you to assign default methods that catch and execute APIs called with these unchecked APIs. Instead of relying on string-based names, it matches against role and signature. 

This lets you build the tooling you need under real-world deployment circumstances.